packages/wrangler/src/index.tsx

```diff
@@ -213,7 +213,7 @@ function demandOneOfOption(...options: string[]) {
 class CommandLineArgsError extends Error {}
 class DeprecationError extends Error {
   constructor(message: string) {
-    super(`DEPRECATION WARNING:\n${message}`);
+    super(`🔴 DEPRECATION WARNING:\n${message}`);
```

Comment:
It seems wrong that this has a different icon to the other deprecation warnings.

```suggestion
    super(`🦺 DEPRECATION:\n${message}`);
```

Reply:
I am fan of that vest, (family construction workers) it means safety/working at the same time in my mind!!